docs/tutorials/moving-pictures/moving-pictures.md

```diff
@@ -0,0 +1,336 @@
+view qurro-plot.qzv` or by uploading the file to
+[view.qiime2.org](https://view.qiime2.org/). (You can also just click on the
+"view" link right above to see a precomputed version of this visualization.)
+
+## Interacting with a Qurro visualization
+
+Let's view `qurro-plot.qzv`, as described above.
+
+<img src="https://raw.githubusercontent.com/biocore/qurro/master/docs/tutorials/moving-pictures/screenshots/qurro1.png" alt="Qurro interface screenshot #1. No features are selected." />
+
+So right away we see two things: on the left a plot of rankings (in this case,
+loadings) for each feature, and on the right a plot of selected features' log
+ratios in samples. In this tutorial these plots will be referred to as the
+__rank plot__ and __sample plot__, respectively.
+
+Since no features are currently selected to be part of a log ratio, these plots
+look pretty empty. So let's select some features!
+
+### Selecting features to construct log ratios
+
+Recall that we'd like to analyze the log ratios between features' abundances.
+In Qurro, "selecting features" lets us define a __log ratio__ between multiple
+features' abundances in each sample. That is, we can select the numerator and
+denominator feature(s) that make up this log ratio.
+
+There are a few ways of selecting features in Qurro:
+
+- One way is just by clicking on the rank plot twice. The first click sets
+  the numerator feature for a log ratio, and the second click sets the
+  denominator feature for the log ratio.
+- You can also select features based on a text-search through their feature
+  IDs or metadata. For example, it's possible to construct the log ratio of all
+  features with taxonomy annotations containing the text `o__Fusobacteriales` over all
+  features with taxonomy annotations containing the text `o__Pseudomonadales`.
+    - This is equivalent to the log ratio of all ranked features identified as
+      being in the order
+      [_Fusobacteriales_](https://en.wikipedia.org/wiki/Fusobacteriales) over all
+      ranked features identified as being in the order
+      [_Pseudomonadales_](https://en.wikipedia.org/wiki/Pseudomonadales).
+    - In this case -- where an arbitrary number of features can be in the numerator
+      and denominator of the log ratio -- the log ratio is computed for a given
+      sample by summing the feature abundances of the numerator features,
+      summing the feature abundances of the denominator features, and then
+      taking the log ratio of these sums. Written out as a formula, this is
+      `log(top sum) - log(bottom sum)` (or, [equivalently](https://en.wikipedia.org/wiki/List_of_logarithmic_identities#Using_simpler_operations),
+      `log(top sum / bottom sum)`).
+- You can also select features by searching through their feature rankings or
+  metadata numerically (e.g. you can select all features with a differential
+  above a certain threshold). This tutorial won't cover this option, but feel
```

Comment:
I think this comma is unnecessary

Reply:
`This tutorial won't cover this option, but feel free ...` -> `This tutorial won't cover this option; feel free ...`